libcodechecker/libhandlers/analyze.py

```diff
@@ -174,6 +174,14 @@ def is_ctu_capable():
                                     "compiler on the system is special, e.g. "
                                     "when doing cross-compilation.")
 
+    analyzer_opts.add_argument('--capture-analysis-output',
+                               action='store_true',
+                               default=False,
+                               required=False,
+                               help="Store the (standard and error) output "
```

Comment:
Perhaps "standard output" and "standard error" would convey better, and also keep us in line with the UNIX Principles.